### PR TITLE
updated the readme

### PR DESCRIPTION
**Pull request by harsh204016**
fix some small mistakes in readme , please review and accept it.

### CODE REVIEW

File: gtrending/fetch.py [hunk 1]
```diff
@@ -77,10 +77,10 @@ def fetch_developers(language: str = "", since: str = "daily") -> dict:
 
 
 def languages_list() -> list:
-    """Fetch languages
+    """Fetch programming languages.
```
Comment (hedyhli) [inline gtrending/fetch.py:80]: If you want the docstring to be this, then please update that part in the readme as well

File: gtrending/fetch.py [hunk 2]
```diff
@@ -54,14 +54,14 @@ def fetch_repos(
 
 
 def fetch_developers(language: str = "", since: str = "daily") -> dict:
-    """Fetch trending developers on GitHub
+    """Fetch the trending developers on GitHub.
```
Comment (hedyhli) [inline gtrending/fetch.py:57]: I think it's a bit extra
```suggestion
    """Fetch trending developers on GitHub.
```

**Review by hedyhli — CHANGES_REQUESTED** [review]
Almost there! Just keep in mind that we want the docstrings in the API be identical to that of the code.

**Comment by harsh204016** [comment]
Hi @hedyhli , I have updated the readme as requested please review and merge it.

**Review by hedyhli — COMMENTED** [review]
Thanks!

**Review by hedyhli — APPROVED** [review]

**Pull request merged**